fuzzing/tools/copy_corpus.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""
+Copys and renames a set of corpus files into a given directory.
+"""
+
+from absl import app
+from absl import flags
+from sys import stderr
+from shutil import copy
```

Comment:
Nit: Don't directly import symbols, as per the style guide: https://google.github.io/styleguide/pyguide.html#22-imports

Just `import shutil` here.